packages/backend-proxy-middleware/package.json

```diff
@@ -14,13 +14,13 @@
     "author": "@SAP/ux-tools-team",
     "main": "dist/index.js",
     "scripts": {
-        "build": "pnpm clean && tsc -p tsconfig-build.json",
-        "clean": "rimraf dist",
+        "build": "tsc --build",
```

Comment:
That was a request by @devinea because we are trying to optimize the builds and only build if something changed, but if we delete the `dist` folder before every build then, well, it always changed.

Reply:
Thanks!